src/OpenSage.Game/Diagnostics/MainView.cs

```diff
@@ -61,15 +61,19 @@ public void Draw(ref bool isGameViewFocused)
                     {
                         foreach (var mapCache in _context.Game.ContentManager.IniDataContext.MapCaches)
                         {
-                            if (ImGui.MenuItem($"{mapCache.DisplayName} ({mapCache.Name})"))
+                            if (mapCache.IsMultiplayer && ImGui.MenuItem($"{mapCache.DisplayName} ({mapCache.Name})"))
```

Comment:
I want all maps to be visible in this menu, otherwise there's not (currently) any way to get to single player maps. If it crashes, we should fix the crash rather than hiding them altogether.